src/components/drawing/index.js

```diff
@@ -512,32 +513,36 @@ drawing.bBox = function(node) {
         return Lib.extendFlat({}, savedBBoxes[saveNum.value]);
     }
 
-    var test3 = d3.select('#js-plotly-tester'),
```

Comment:
Nicely done. There's only one tester div per page, so might as well cache in the `Drawing` module object. :+1: